packages/components/src/entities/SamplesDeriveButton.tsx

```diff
@@ -22,57 +23,42 @@ export interface SamplesDeriveButtonProps
 
 export const SamplesDeriveButton: FC<SamplesDeriveButtonProps> = memo(props => {
     const { model, asSubMenu, ...createSampleMenuProps } = props;
-    const [selectionsAreSet, setSelectionsAreSet] = useState<boolean>(false);
-    const [selectionData, setSelectionData] = useState<Map<any, any>>();
-    const selectedCount = useMemo(() => model?.selections?.size ?? -1, [model?.selections]);
-    const useSelectionData = useMemo(() => model?.filterArray.length > 0 && selectedCount <= MAX_EDITABLE_GRID_ROWS , [model?.filterArray]);
+    const { filterArray, isLoadingSelections, schemaQuery, selections } = model;
```

Comment:
@labkey-susanh Do you know why we care about filters being present for this loading of selected data? Couldn't users want to derive samples from a selection where there is no filters applied?

Reply:
I believe this came from the need to distinguish whether to use snapshot selections or not (related to [this PR.](https://github.com/LabKey/labkey-ui-components/pull/1046)) and [Issue 46465](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=46465). 

Without the filter, we don't need to use a snapshot selection.  @cnathe updated the logic slightly related to when and how we decide about snapshot selections, so perhaps this is now obsolete.